app/src/main/java/com/main/mealplanner/dto/MealPlan.kt

```diff
@@ -8,6 +8,6 @@ import java.util.*
 @Entity(tableName="mealplan")
 data class MealPlan(var CookSchedule: Date, var RecipeId : String, var OwnerEmail : String, @PrimaryKey @SerializedName("id") var MealPlanId:Int = 0) {
     override fun toString(): String {
-        return "Cook Recipe #" + RecipeId + " at " + CookSchedule.toString()
+        return "Cook Recipe #$RecipeId at $CookSchedule"
```

Comment:
I know Android help with the refactor but I  also I based on https://kotlinlang.org/docs/classes.html formatted their string

Reply:
Oh, yeah.  Great use of Kotlin string utilities.  Much cleaner (and more efficient) than concatenation!